bin/index.js

```diff
@@ -64,36 +64,12 @@ const askQuestions0 = () => {
 
 const askQuestions1 = () => {
   const questions = [
-    {
-      name: "_publicKey",
-      type: "input",
-      message: "Public Key?",
-      default: publicKey
-    },
-    {
-      name: "_secretKey",
-      type: "input",
-      message: "Secret Key?",
-      default: secretKey
-    },
     {
       name: "_cipJsonFilePath",
```

Comment:
I think this question is also redundant. As far as I understand we ask here for the off-chain metadata file path (later on the root hash is calculated based on this file). If so, then this question does not make sense, since later we ask for the metadata url which is the url of the same off-chain data. We should rather fetch the off-chain metadata from received link and calculate the root hash based on the fetched content. Shouldn't it work this way? 
![image](https://github.com/input-output-hk/dapp-store-cip72/assets/9270461/73107ec5-0847-4f2a-b388-42923743315b)

Reply:
I don't agree with removing this question. How about using the local metadata the way it is. A lot can go wrong with a URL. First, it could be a non-json file, the URL path could be wrong, the user might prefer to upload the local version instead of the uploaded version. I think for a first version, this works. We can improve this when we are out of the MVP phase instead of adding more complexity to this.

Reply:
We agreed there are issues with this but we postpone improvements. @greatertomi will create a task for that.